project/Dependencies.scala

```diff
@@ -54,25 +55,15 @@ object Dependencies {
   val scalaCheckVersion = "1.15.1"
 
   val Versions =
-    Seq(
-      crossScalaVersions := Seq(scala212Version, scala213Version),
-      scalaVersion := {
-        System.getProperty("akka.build.scalaVersion", "default") match {
-          case v if v.startsWith("2.13") => scala213Version
-          case v if v.startsWith("2.12") => scala212Version
-          case v if v.startsWith("3.")   => scala3Version
-          case "default"                 => scala213Version
-        }
-      },
-      java8CompatVersion := {
-        CrossVersion.partialVersion(scalaVersion.value) match {
-          // java8-compat is only used in a couple of places for 2.13,
-          // it is probably possible to remove the dependency if needed.
-          case Some((3, _))            => "1.0.0"
-          case Some((2, n)) if n >= 13 => "1.0.0"
-          case _                       => "0.8.0"
-        }
-      })
+    Seq(crossScalaVersions := allScalaVersions, scalaVersion := allScalaVersions.head, java8CompatVersion := {
+      CrossVersion.partialVersion(scalaVersion.value) match {
+        // java8-compat is only used in a couple of places for 2.13,
+        // it is probably possible to remove the dependency if needed.
+        case Some((3, _))            => "1.0.0"
+        case Some((2, n)) if n >= 13 => "1.0.0"
+        case _                       => "0.8.0"
```

Comment:
if only used for 2.13 code base, why we pick 0.8.0 for 2.12? This has caused so much trouble already. 

I guess there is a reason for sticking to 0.8.0. Does anyone know it? Could we make add a comment here?